kubernetes/README.md

```diff
@@ -37,7 +37,8 @@ The kubernetes check is compatible with all major platforms
 See [metadata.csv](https://github.com/DataDog/integrations-core/blob/master/kubernetes/metadata.csv) for a list of metrics provided by this integration.
 
 ### Events
-The Kubernetes check does not include any event at this time.
+
+Since dd-agent 5.17, [enable the agent leader election option](#how-to-elect-a-leader-agent-in-order-to-avoid-duplicated-events) in order to gather events from your kubernetese nodes without any duplicated events.
```

Comment:
As the 5.17.0 release, Datadog Agent now supports built in [leader election option](#how-to-elect-a-leader-agent-in-order-to-avoid-duplicated-events) for the Kubernetes event collector. Once enabled, users no longer need to deploy an additional event collection container to their cluster. Instead agents will coordinate to ensure only one agent instance is gathering events at a given time.

Reply:
:+1: